package.json

```diff
@@ -1,43 +1,68 @@
 {
   "name": "coronadatascraper",
   "version": "1.0.0",
-  "description": "A scraper for Cornavirus data",
+  "description": "A scraper for Coronavirus data",
   "main": "cli.js",
   "author": "Larry Davis <lazdnet@gmail.com>",
   "license": "BSD",
   "type": "module",
   "scripts": {
-    "postinstall": "node initSSL.js",
-    "timeseries": "node timeseries.js",
-    "start": "NODE_OPTIONS='--insecure-http-parser' node cli.js",
-    "winStart": "set NODE_OPTIONS='--insecure-http-parser' && node cli.js",
-    "updateModules": "git submodule update --remote",
-    "update": "npm run updateModules && rm -rf cache/* && npm run start",
     "build": "yarn start && yarn timeseries && yarn buildSite",
     "buildSite": "cp -r site/* dist/ && ./tools/copySpectrumCSS.sh",
-    "deploy": "npm run dateData && npm run deploySite",
     "dateData": "DATE=$(date +'%Y-%m-%d') && cp dist/data.json dist/data-$DATE.json && cp dist/features.json dist/features-$DATE.json && cp dist/data.csv dist/data-$DATE.csv",
+    "deploy": "npm run dateData && npm run deploySite",
     "deploySite": "npm run buildSite && gh-pages -a -d dist/ -e .",
-    "test": "jest"
+    "lint": "eslint .",
+    "postinstall": "node initSSL.js",
+    "start": "NODE_OPTIONS='--insecure-http-parser' node cli.js",
+    "test": "jest",
+    "timeseries": "node timeseries.js",
+    "update": "npm run updateModules && rm -rf cache/* && npm run start",
+    "updateModules": "git submodule update --remote",
+    "winStart": "set NODE_OPTIONS='--insecure-http-parser' && node cli.js"
   },
   "dependencies": {
     "@adobe/spectrum-css": "^2.18.0",
     "@turf/turf": "^5.1.6",
     "cheerio": "^1.0.0-rc.3",
     "csv": "^5.3.2",
+    "csv-parse": "^4.8.8",
+    "csv-stringify": "^5.3.6",
```

Comment:
We can remove `csv` itself if we add these as deps directly

Reply:
IDM, which is better?

Reply:
I say drop `csv`, thanks!